src/__experimental__/components/tile/tile.component.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import { StyledTile, TileContent } from './tile.style.js';
+
+const Tile = (props) => {
+  const {
+    as,
+    children,
+    padding,
+    orientation,
+    width,
+    ...rest
+  } = props;
+
+  const wrappedChildren = React.Children.map(children, (child, index) => {
+    const { width: contentWidth, ...childProps } = child.props;
+    const key = child.key || `tile-content-${index + 1}`;
+
+    return (
+      <TileContent
+        key={ key }
+        width={ contentWidth }
+      >
+        {React.cloneElement(child, childProps)}
+      </TileContent>
+    );
+  });
+
+  return (
+    <StyledTile
+      padding={ padding }
+      tileTheme={ as }
+      orientation={ orientation }
+      width={ width }
+      { ...rest }
+      { ...tagComponent('tile', props) }
+    >
+      {wrappedChildren}
```

Comment:
Did this pass linting? It'll probably complain about spaces inside braces.

Reply:
Yep, passed fine - I've noticed whenever I'm passing a child in curly braces like this it forces the formatting so that there are no spaces. Always thought it was odd, as it's at odds with the rule enforced elsewhere, but it automatically changes it for me if I try and add the spaces, so I've left it.

Reply:
There's a ticket on the backlog to investigate some of the issues we seem to be having with eslint, I'll add this as an example to the ticket. I suspect it's the `"eslint.autoFixOnSave": true` rule in `settings.json` isn't picking up this edge case for some reason